### ISSUE TITLE
Some error after install (need help)

### ISSUE THREAD

**Issue by geds3169** [Domain-Connect/DomainConnectDDNS-Python#35]
Hello,
I wanted to thank you first for this tool.

I have a problem, I used your solution with IONOS, the executable worked fine on Windows 10. I invested in a server running Windows Server 2019, there I couldn't make it work.

So I tried on my VM's under Windows 10 that no longer works, under Debian Buster I also have problems.

Can you help me

Some screen

Python 2.7.16 running and 3.7 installed 
pip 18.1

[https://ibb.co/CJKBDDm](url)

forced 3.7

and reinstall same problems 

[https://ibb.co/34KKmtP](url)

Once again thank you for your work and thank you for excusing my English. If you can help me, I would appreciate it.

cordially

**Comment by pawel-kow** [comment]
Hi,

For python 2.7 it looks like `parse` library is missing. I'd need to take a look in spare time why it is like that.
For python 3.7 all is OK with the tool. Your domain is missing `_domainconnect` record, which is put by IONOS by default.
You must have deleted it, please see issue #33 for a solution.

**Comment by geds3169** [comment]
Thanks for taking the time to answer me, I'll take a look at this and get back to you.

thanks again


Provenance : Courrier pour Windows 10

De : Pawel Kowalik
Envoyé le :mardi 8 septembre 2020 09:15
À : Domain-Connect/DomainConnectDDNS-Python
Cc : geds3169; Manual
Objet :Re: [Domain-Connect/DomainConnectDDNS-Python] Some error afterinstall (need help) (#35)

Closed #35.

**Comment by geds3169** [comment]
Hi again,

I just installed dnsutils in order to have the dig command. Without having made any modification on my recordings, the command dig TXT doman.web-connectivity.fr seems to return the correct parameters. Do you have another lead?

Thank you

Cordialy Guilhem

Provenance : Courrier pour Windows 10

De : Pawel Kowalik
Envoyé le :mardi 8 septembre 2020 09:15
À : Domain-Connect/DomainConnectDDNS-Python
Cc : geds3169; Manual
Objet :Re: [Domain-Connect/DomainConnectDDNS-Python] Some error afterinstall (need help) (#35)

Closed #35.

**Comment by pawel-kow** [comment]
@geds3169 
```
root@localhost:~# dig TXT _domainconnect.web-connectivity.fr

; <<>> DiG 9.10.3-P4-Ubuntu <<>> TXT _domainconnect.web-connectivity.fr
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 4816
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;_domainconnect.web-connectivity.fr. IN TXT

;; AUTHORITY SECTION:
web-connectivity.fr.    589     IN      SOA     ns1035.ui-dns.biz. hostmaster.1und1.com. 2017060297 28800 7200 604800 600

;; Query time: 1 msec
;; SERVER: 212.227.123.16#53(212.227.123.16)
;; WHEN: Tue Sep 08 09:34:50 UTC 2020
;; MSG SIZE  rcvd: 136
```
